tests/client/oqs-generate-test-id-keys.sh

```diff
@@ -29,6 +29,6 @@ for value in \
 do	
 	if [ ! -f ../keys/id_${value} ]; then
 		echo "Generating keypair id_${value}."
-		ssh-keygen -t ${value/-/_} -q -N "" -f ../keys/id_${value} -C bob@bob.com || exit 1
+		ssh-keygen -t ${value//-/_} -q -N "" -f ../keys/id_${value} -C bob@bob.com || exit 1
```

Comment:
Does this fix an escape issue? Doesn't look like a v8 update thing.

Reply:
It is related to the v8 update. The additional forward slash replaces all instances of the pattern, whereas without it, only the first instance is replaced. Before this wasn't an issue, but in v8, some of the ciphersuites have multiple hyphens, mainly because what used to be called p{256,384,521}-<pq alg> are now ecdsa-nistp{256,384,521}-<pq alg>. So without this fix, the filenames being generated for the keys ended up being of the form "id_ecdsa_nistp256-picnicL1FS", for example, where we really wanted all of those hyphens to change to underscores.